src/Shouldly/Shouldly.csproj

```diff
@@ -1,23 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
     <TargetFrameworks>net451;netstandard1.3;netstandard2.0</TargetFrameworks>
-    <AssemblyName>Shouldly</AssemblyName>
-    <PackageId>Shouldly</PackageId>
-    <PackageTags>test;unit;testing;TDD;AAA;should;testunit;rspec;assert;assertion;framework</PackageTags>
-    <PackageIconUrl>https://raw.githubusercontent.com/shouldly/shouldly/master/assets/logo_128x128.png</PackageIconUrl>
-    <PackageProjectUrl>http://shouldly.github.com</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/shouldly/shouldly/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <LangVersion>latest</LangVersion>
+    <Optimize>false</Optimize>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <AssemblyOriginatorKeyFile>sn.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  
+    <Description>Shouldly - Assertion framework for .NET. The way asserting *Should* be</Description>
+    <Authors>Jake Ginnivan, Joseph Woodward</Authors>    
```

Comment:
Is this right?

Reply:
Yes, that's right. Though I feel it should be everyone who has contributed in there personally. It always feels odd distilling a large OSS contribution such as this down to two names.